pipelines/broad/rna_seq/RNAWithUMIsPipeline.changelog.md

```diff
@@ -1,3 +1,8 @@
+#1.0.10
```

Comment:
you need to update the `pipeline_version` variable in the WDL as well:
https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl#L23